gke-debian-openjdk/src/main/docker/Dockerfile

```diff
@@ -17,44 +17,25 @@ ENV DEBIAN_FRONTEND noninteractive
 
 # Update debian 
 RUN echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list \
- && apt-get -q update \
- && apt-get -y -q --no-install-recommends install \
-    ca-certificates \
-    openjdk-8-jre \
+  && apt-get -q update \
+  && apt-get -y -q --no-install-recommends install \
+    ca-certificates-java=20161107'*' \
+    openjdk-8-jre-headless=8u121'-*' \
     netbase \
     wget \ 
     unzip \
  && apt-get clean \
  && rm /var/lib/apt/lists/*_*
 
-# workaround for https://bugs.debian.org/775775
```

Comment:
I would leave all of this old code below, just to minimize the risk of this PR. The only lines we need changed are 22 and 23 above.